init.sh

```diff
@@ -1,11 +1,4 @@
-#!/bin/bash
-
-# vars for jupyter
+source ~/.bash_profile
 storage_dir=$(readlink -f $PWD)
 export TCHANNEL_BASE=${storage_dir}
-export JUPYTER_PATH=${storage_dir}/.jupyter
-export JUPYTER_RUNTIME_DIR=${storage_dir}/.local/share/jupyter/runtime
-export JUPYTER_DATA_DIR=${storage_dir}/.local/share/jupyter
-export IPYTHONDIR=${storage_dir}/.ipython
```

Comment:
These aren't needed either?

Reply:
I didn't want to think about Jupyter at the moment but maybe I should?